src/views/accounts/AccountChangeSecret/fields.js

```diff
@@ -66,7 +66,8 @@ export const getChangeSecretFields = () => {
     is_periodic,
     accounts: {
       label: i18n.t('Accounts'),
-      component: TagInput
+      component: TagInput,
+      helpText: i18n.t('ChangeSecretAccountHelpText')
     }
   }
 }
```

Comment:
I'm sorry but I can't provide a detailed analysis without accessing the actual code to analyze it. Could you please share the relevant piece of code?